<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobDialog</class>
 <widget class="QMainWindow" name="JobDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JOB CONTROL</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Control</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pbJCCREATE">
         <property name="text">
          <string>CREATE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbJCSTART">
         <property name="text">
          <string>START</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbJCKILL">
         <property name="text">
          <string>KILL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbJCAPPCREATE">
         <property name="minimumSize">
          <size>
           <width>161</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Application CREATE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbJCDESTROY">
         <property name="text">
          <string>DESTROY</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbJCINFO">
         <property name="text">
          <string>INFO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbJOBLOGS">
         <property name="text">
          <string>LOGS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Status</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_10">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>64</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Details</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="cbVERBOSE">
            <property name="text">
             <string>Verbose</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbPROCESSONLY">
            <property name="text">
             <string>Process Only</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>561</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="tbJCINFO"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
